packages/react-components/.storybook/fluentuiTheme.js

```diff
@@ -0,0 +1,37 @@
+import { create } from '@storybook/theming';
```

Comment:
copied from `react-examples/.storybook`

Reply:
Instead of duplicating the custom styling files, let's just delete them from react-examples since having the customized storybook styling there isn't important.

Reply:
I'll address those in PR followup 
![2021-08-13 at 10 42](https://user-images.githubusercontent.com/1223799/129330111-a5695a96-e73d-4ee2-9c15-d36e4f21861d.png)

Reply:
It would have been nice to do that together with this change so it would show up as a rename not a new file (at least in git tools that are configured to handle renames)